### PR TITLE
fix #11

### LINKED ISSUE #11: Select tag not displayed properly
Fix could be in survey/templates/survey/question.html displaying form directly instead of itering on field.

It's also possible to use bootstrap-form directly to render form.

### PR DESCRIPTION
**Pull request by raonyguimaraes**
This fixes the select tags in the Survey.

### CODE REVIEW

File: survey/templates/survey/question.html [hunk 1]
```diff
@@ -9,22 +9,7 @@
     </thead> -->
     <tbody>
 {% for form in response_form %}
-	{% if form.field.widget.attrs.category == category.name or not form.field.widget.attrs.category %}
```
Comment (Pierre-Sassoulas) [inline survey/templates/survey/question.html]: This need to stay or every category will contain every question.

File: survey/templates/survey/question.html [hunk 2]
```diff
@@ -9,22 +9,7 @@
...
-        <tr class="{% if form.errors%} danger {% endif %}">
-            <td>
-	            {{ form.label|safe }}
-	            {% if form.field.required %}
-	                <span class="glyphicon glyphicon-asterisk" style="color:red"> </span>
```
Comment (Pierre-Sassoulas) [inline survey/templates/survey/question.html]: This needs to stay or the user won't be noticed when a question is required.

File: survey/templates/survey/question.html [hunk 3]
```diff
@@ -9,22 +9,7 @@
     </thead> -->
     <tbody>
 {% for form in response_form %}
-	{% if form.field.widget.attrs.category == category.name or not form.field.widget.attrs.category %}
-        <tr class="{% if form.errors%} danger {% endif %}">
-            <td>
-	            {{ form.label|safe }}
```
Comment (Pierre-Sassoulas) [inline survey/templates/survey/question.html]: Maybe replace `safe` by `bootstrap` here and see how it goes ?

**Comment by Pierre-Sassoulas** [comment]
@florianmouchantaf I think you will get a better solution to your problem with what was done here : https://github.com/Pierre-Sassoulas/django-survey/pull/49.

**Comment by florianmouchantaf** [comment]
Thanks for the help!!

On 20 Sep 2019 21:25, Pierre Sassoulas <notifications@github.com> wrote:

@florianmouchantaf<https://github.com/florianmouchantaf> I think you will get a better solution to your problem with what was done here : #49<https://github.com/Pierre-Sassoulas/django-survey/pull/49>.

**Comment by Pierre-Sassoulas** [comment]
Closed, because it was fixed in #49

**Pull request closed without merging**